packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -10,8 +10,59 @@ function setDefaultCustomization (JSONEditor) {
 
   editorsKeys.forEach(key => {
     JSONEditor.defaults.editors[key] = class Customization extends JSONEditor.defaults.editors[key] {
+      showEditJSON(){
+        super.showEditJSON()
+        if (this.editjson_holder && this.editjson_textarea && this.editjson_control) {
+          const outsideClickListener = event => {
+            if (!this.editjson_holder.contains(event.target) && this.editjson_holder.style.display !== 'none') {
+              this.hideEditJSON()
+              removeClickListener()
+            }
+          }
+          const removeClickListener = () => {
+            document.removeEventListener('click', outsideClickListener)
+          }
+          document.addEventListener('click', outsideClickListener)
+          this.editjson_textarea.style.width = '450px'
+          this.editjson_textarea.style.height = '340px'
+          this.editjson_holder.style.left = `${parseInt(this.editjson_holder.style.left, 10) -this.editjson_holder.offsetWidth - parseInt(this.editjson_control.style.marginLeft, 10)}px`
+          this.editjson_holder.style.top = `-104px`
+        }
+      }
       postBuild() {
         super.postBuild()
+        if (this.editjson_holder && this.editjson_textarea && this.editjson_copy && this.editjson_save && this.editjson_cancel) {
+          this.editjson_inline_header = document.createElement('div')
+          this.editjson_inline_header.style.display = 'flex'
```

Comment:
Can't you do:

```
this.editjson_inline_header.style = {
  display: 'flex',
  ...
}
```

Reply:
I tried, but actually style properties are not passed correctly to elements if done in that way!